app/models/form/lettings/questions/created_by_id.rb

```diff
@@ -8,21 +10,27 @@ def initialize(id, hsh, page)
   end
 
   def answer_options
-    answer_opts = { "" => "Select an option" }
-    return answer_opts unless ActiveRecord::Base.connected?
+    return ANSWER_OPTS unless ActiveRecord::Base.connected?
 
-    User.select(:id, :name, :email).each_with_object(answer_opts) do |user, hsh|
+    User.select(:id, :name, :email).each_with_object(ANSWER_OPTS.dup) do |user, hsh|
       hsh[user.id] = "#{user.name} (#{user.email})"
       hsh
     end
   end
 
-  def displayed_answer_options(log, _user = nil)
-    return answer_options unless log.owning_organisation && log.managing_organisation
+  def displayed_answer_options(log, user = nil)
+    return ANSWER_OPTS unless user
+    return ANSWER_OPTS unless user.support? || user.data_coordinator?
 
     user_ids = [""]
-    user_ids += log.owning_organisation.users.pluck(:id)
-    user_ids += log.managing_organisation.users.pluck(:id)
+    user_ids += if user.support?
+                  [
+                    (log.owning_organisation&.users&.pluck(:id) if log.owning_organisation),
+                    (log.managing_organisation&.users&.pluck(:id) if log.managing_organisation),
+                  ]
+                else
+                  user.organisation.users.pluck(:id)
```

Comment:
What is the data coordinator want to assign to someone from managing organisation? Should they be able to do that?